### PR TITLE
doc: update changelogs for NCS 1.7.0

### PR DESCRIPTION
**Pull request by greg-fer**
Changed master branch heading to nRF Connect SDK 1.7.0.
NCSDK-11022

Signed-off-by: Grzegorz Ferenc <Grzegorz.Ferenc@nordicsemi.no>

### CODE REVIEW

**Comment by greg-fer** [comment]
@czeslawmakarski , @FrancescoSer , no changes needed for the 802.15.4 changelog?

**Review by rugeGerritsen — APPROVED** [review]

File: zboss/CHANGELOG.rst [hunk 1]
```diff
@@ -7,19 +7,20 @@ Changelog
...
 
-All notable changes to this project in |NCS| are documented in this file.
+All notable changes to this project in the |NCS| are documented in this file.
 
+nRF Connect SDK v1.7.0
+**********************
 
-Master branch
-*************
+All the notable changes included in the nRF Connect SDK v1.7.0 release are documented in this section.
```
Comment (umapraseeda) [inline zboss/CHANGELOG.rst:15]:
```suggestion
All the notable changes included in the |NCS| v1.7.0 release are documented in this section.
```

File: softdevice_controller/CHANGELOG.rst [hunk 2]
```diff
@@ -9,10 +9,10 @@ Changelog
 
 All the notable changes to this project are documented in this file.
 
-Master branch
-*************
+nRF Connect SDK v1.7.0
+**********************
 
-All the notable changes added to the master branch are documented in this section.
+All the notable changes included in the nRF Connect SDK v1.7.0 release are documented in this section.
```
Comment (umapraseeda) [inline softdevice_controller/CHANGELOG.rst:15]:
```suggestion
All the notable changes included in the |NCS| v1.7.0 release are documented in this section.
```

File: nfc/CHANGELOG.rst [hunk 3]
```diff
@@ -9,8 +9,10 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
-master
-******
+nRF Connect SDK v1.7.0
+**********************
+
+All the notable changes included in the nRF Connect SDK v1.7.0 release are documented in this section.
```
Comment (umapraseeda) [inline nfc/CHANGELOG.rst:15]:
```suggestion
All the notable changes included in the |NCS| v1.7.0 release are documented in this section.
```

File: mpsl/CHANGELOG.rst [hunk 4]
```diff
@@ -9,10 +9,10 @@ Changelog
 
 All the notable changes to this project are documented in this file.
 
-Master branch
-*************
+nRF Connect SDK v1.7.0
+**********************
 
-All the notable changes added to the master branch are documented in this section.
+All the notable changes included in the nRF Connect SDK v1.7.0 release are documented in this section.
```
Comment (umapraseeda) [inline mpsl/CHANGELOG.rst:15]:
```suggestion
All the notable changes included in the |NCS| v1.7.0 release are documented in this section.
```

**Review by umapraseeda — APPROVED** [review]

**Comment by thomasstenersen** [comment]
FYI: There's a couple few bugfixes in the SoftDevice controller changelog that needs to be removed from the known issues in sdk-nrf:
- DRGN-15586: “HCI LE Set Scan Parameters” accepts a scan window greater than the scan interval
- DRGN-13594: The channel map provided by the LE Host Set Channel Classification HCI command is not always applied on the secondary advertising channels

**Review by totyz — APPROVED** [review]

**Comment by greg-fer** [comment]
> FYI: There's a couple few bugfixes in the SoftDevice controller changelog that needs to be removed from the known issues in sdk-nrf:

@thomasstenersen , these are not marked for 1.7.0 in the known issues PR, so I guess we're fine?

**Review by czeslawmakarski — APPROVED** [review]

**Comment by rugeGerritsen** [comment]
> > FYI: There's a couple few bugfixes in the SoftDevice controller changelog that needs to be removed from the known issues in sdk-nrf:
> 
> @thomasstenersen , these are not marked for 1.7.0 in the known issues PR, so I guess we're fine?

@greg-fer , @thomasstenersen , those were addressed in https://github.com/nrfconnect/sdk-nrf/pull/5475#pullrequestreview-743554178 :)

File: softdevice_controller/CHANGELOG.rst [hunk 5]
```diff
@@ -43,7 +43,7 @@ Bug fixes
 Changes
```
Comment (seko-nordic) [inline softdevice_controller/CHANGELOG.rst:43]: this section is here second time, first at line 23.

**Review by seko-nordic — APPROVED** [review]

**Review by b-gent — APPROVED** [review]

**Review by anangl — APPROVED** [review]

**Review by FrancescoSer — APPROVED** [review]

File: nrf_802154/doc/CHANGELOG.rst [hunk 6]
```diff
@@ -9,6 +9,30 @@ Changelog
...
+  It now allows specifying whether to encrypt or inject dynamic data into the outgoing frame, or do both.
+  The :c:type:`nrf_802154_transmitted_frame_props_t` type is used for this purpose.
+
+Bug fixes
+=========
+
+* Fixed an issue where it would not be possible to transmit frames with invalid Auxiliary Security Header if :kconfig:`CONFIG_NRF_802154_ENCRYPTION` was set to ``n``. (KRKNWK-11218)
+* Fix an issue with the IE Vendor OUI endianness. (KRKNWK-10633)
+* Fixed various bugs in the MAC Encryption layer. (KRKNWK-10646)
+
```
Comment (ankuns) [inline nrf_802154/doc/CHANGELOG.rst:35]: Please add following
```
Limitations
===========

* Application and device drivers (excluding those compliant with MPSL) must not use IRQ priority higher than NRF_802154_SWI_PRIORITY and NRF_802154_SL_RTC_IRQ_PRIORITY.

```

**Review by ankuns — CHANGES_REQUESTED** [review]

**Review by ankuns — APPROVED** [review]

File: nrf_802154/doc/CHANGELOG.rst [hunk 7]
```diff
@@ -9,6 +9,35 @@ Changelog
...
+Limitations
+===========
+
+* Application and device drivers (excluding those compliant with :ref:`nrfxlib:mpsl`) must not use IRQ priority higher than :c:macro:`NRF_802154_SWI_PRIORITY` and :c:macro:`NRF_802154_SL_RTC_IRQ_PRIORITY`.
+
```
Comment (czeslawmakarski) [inline nrf_802154/doc/CHANGELOG.rst:40]:
```suggestion
Known issues
============

* At the rare occasions during the use of BLE and 802.15.4 multiprotocol the device might assert. (KRKNWK-11384)
* Transmitting the 802.15.4 frame with the improperly populated Auxilliary Security Header might result in assert. (KRKNWK-11204)
```

**Review by czeslawmakarski — CHANGES_REQUESTED** [review]
Known issues for 802.15.4 Radio Driver

File: nrf_802154/doc/CHANGELOG.rst (hunk 7 above)
Comment (greg-fer) [reply nrf_802154/doc/CHANGELOG.rst:40]: We don't track known issues outside of sdk-nrf. This cannot be added here. Add a comment to: https://github.com/nrfconnect/sdk-nrf/pull/5475

Comment (greg-fer) [reply nrf_802154/doc/CHANGELOG.rst:42]: @czeslawmakarski , please re-review.

**Review by czeslawmakarski — APPROVED** [review]

**Review by shantha-14 — APPROVED** [review]

**Review by carlescufi — APPROVED** [review]

**Review by gmarull — APPROVED** [review]

**Pull request merged**